test/legacy-upgrade/create-in-v0.27.0-object-with-avg.td

```diff
@@ -0,0 +1,15 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
```

Comment:
why is the test create-in-v0.27.0? I would have guessed v0.65.0

Reply:
I think I don't fully understand how these three tests work together to test this

Reply:
We use `v0.27.0` for the first create-in file because that designates the _earliest_ version at which we can create these objects. And in that file we're creating views with the `AVG(...)` function, which we expect to get migrated to `AVG_INTERNAL_V1(...)`. We skip running this file for any version >= v0.66.0 though and instead have a second version `create-in-v0.66.0-object-with-avg`, that creates the same views, but manually specifies `AVG_INTERNAL_V1(...)`.

Then we have a `check-in-v0.66` file which will check that in any version >= v0.66 the objects we created use `AVG_INTERNAL_V1(...)`.

The reason for the two `create-in` files is to make sure the `check-in` file will always pass, regardless of the versions we're migrating from. For example, say we run `create-in-v0.27.0-object-with-avg` in v0.67, that will create a view with a `AVG(...)` function, but we _do not_ expect it to get migrated since we're creating it in v0.67. But the `check-in` test case still expects to find the migrated function, hence the `create-in-v0.66` file which manually creates the view with `AVG_INTERNAL_V1(...)`

Hopefully that makes sense? I'm happy to chat more if need be!

Reply:
Thanks for explaining, makes sense!